### PR TITLE
Native token -> TF Token (Ex: ibc denom)

### PR DESCRIPTION
**Pull request by reecepbcups**
Requests by YFoundry. Still needs cleanup work

Also going to remove balance option from other contract, makes things too complex for little upside

### CODE REVIEW

File: artifacts/checksums_intermediate.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
-6c3357b2c43a7642e1f26d7b41f670bcce21580230d9d5c32beaa2b346933e73  ./target/wasm32-unknown-unknown/release/cw20_migrate.wasm
-bd9eddee0a402f80fb1aeb299e989caf8d41093d6d323972a6244ae9c4dba1d2  ./target/wasm32-unknown-unknown/release/tf_example.wasm
-04bc7058a17f3cd78bc03e934d3d9acaf316a10fa1c3b9f2b7c27fffe16b6437  ./target/wasm32-unknown-unknown/release/tokenfactory_core.wasm
+c608d09c12717541f1012694438a38a936180b1995613914ead86749cf8c3939  ./target/wasm32-unknown-unknown/release/cw20_migrate.wasm
+307de97b8c787c64be4513a6a6b1b42bd241ee06cf5247f9bf8409311a9e0ae6  ./target/wasm32-unknown-unknown/release/native_migrate.wasm
+cdcf5bb4a969ae04a7444a6f68aea60d873cd11948f1e7f259311346134e1dd1  ./target/wasm32-unknown-unknown/release/tf_example.wasm
+603160c19bdcad30c339b238f5c16aff6bc0d10528ac04a5908968cd55ba2fb4  ./target/wasm32-unknown-unknown/release/tf_migrate.wasm
+568da005df5461190f4c718ab1a067ae09002ef78727520c7af65962be177fe1  ./target/wasm32-unknown-unknown/release/tokenfactory_core.wasm
```
Comment (JakeHartnell) [inline artifacts/checksums_intermediate.txt]: Maybe we only include these artifacts with releases?

File: contracts/cw20_migrate/Cargo.toml [hunk 2]
```diff
@@ -56,7 +56,8 @@ strum_macros = "0.24"
 
 
 # middleware minting contract
-tokenfactory-types = { git = "https://github.com/Reecepbcups/tokenfactory-core-contract" }
+# tokenfactory-types = { git = "https://github.com/CosmosContracts/tokenfactory-contracts" }
```
Comment (JakeHartnell) [inline contracts/cw20_migrate/Cargo.toml]:
```suggestion
```

**Comment by reecepbcups** [comment]
Going to merge the 2 together (cw20 migrate + native-migrate). Since most logicis really similar & makes testing easier

**Review by JakeHartnell — APPROVED** [review]
Nice!

**Pull request merged**